guides/Version_Control/Git and PR.md

```diff
@@ -0,0 +1,14 @@
+Git has been one of the most famous Version control application in this century.
+It works like taking screenshot of project which you are working on and storing it like every version control.
+Git works on local machine and remote servers like github.
+we follow a particular protocol in using git
+**Pull requests**
+    (Basic)(working only in master branch)
+    1. It is a kind of request you make to the owner of the project when you have contributed to his/her project and want that contribution to be shown.
```

Comment:
Your explanation is really great but could you Git SLOC (Lines of code) for operations pulling a PR and committing it?
Include the SLOC and this PR is good to be merged.